src/LanguageManager/LanguageManagerSettingsPageView.php

```diff
@@ -0,0 +1,96 @@
+<?php # -*- coding: utf-8 -*-
+
+declare( strict_types = 1 );
+
+namespace Inpsyde\MultilingualPress\LanguageManager;
+
+use Inpsyde\MultilingualPress\Asset\AssetManager;
+use Inpsyde\MultilingualPress\Common\Admin\SettingsPageView;
+use Inpsyde\MultilingualPress\Common\HTTP\ServerRequest;
+use Inpsyde\MultilingualPress\Common\Nonce\Nonce;
+
+use function Inpsyde\MultilingualPress\nonce_field;
+
+/**
+ * Language manager settings page view.
+ *
+ * @package Inpsyde\MultilingualPress\LanguageManager
+ * @since   3.0.0
+ */
+final class LanguageManagerSettingsPageView implements SettingsPageView {
+
+	const CONTENT_DISPLAY = 'display_language_manager';
```

Comment:
We usually prefix hook name **constants** with `ACTION_` and `FILTER_`, repsectively. The hook name **value** itself should have the `multilingualpress.` prefix then.